specification/schemas/HookLog.json

```diff
@@ -0,0 +1,121 @@
+{
+  "type": "object",
+  "required": [
+    "attributes",
+    "relationships"
+  ],
+  "properties": {
```

Comment:
Where are the `type` and `id` properties?? 

![image](https://user-images.githubusercontent.com/32866117/58184504-8097d300-7cb1-11e9-92c9-6a61f4c4ec05.png)